onnx/checker.h

```diff
@@ -27,12 +28,27 @@ class ValidationError : public std::runtime_error {
 #define fail_check(...) \
   throw onnx::checker::ValidationError(onnx::MakeString(__VA_ARGS__));
 
+struct CheckerContext {
+  int ir_version;
+  std::unordered_map<std::string, int> opset_imports;
+  int get_ir_version() { return ir_version; }
+  void set_ir_version(int v) { ir_version = v; }
+  // NB: Does a copy
+  std::unordered_map<std::string, int> get_opset_imports() {
+    return opset_imports;
+  }
+  void set_opset_imports(std::unordered_map<std::string, int> imps) {
```

Comment:
Looks like we're not using these setters. We may expose a mutable_opset_imports API instead if you want to and make the two fields private :).